Servers/WindowServer/WSMenu.cpp

```diff
@@ -1,5 +1,5 @@
 /*
- * Copyright (c) 2018-2020, Andreas Kling <kling@serenityos.org>
+ * Copyright (c) 2018-2020, Andreas Kling <kling@serenityos.org>, Shannon Booth <shannon.ml.booth@gmail.com>
```

Comment:
Please add yourself on a new line underneath instead, or this will quickly get out of hand :)